apm-dist/bin/stopCollectorService.sh

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/usr/bin/env sh
+SpringBoot='org.apache.skywalking.apm.collector.boot.CollectorBootStartUp'
```

Comment:
I think you should make this variable more sense.  such as `COLLECTOR_PROCESS_TIPS`

Reply:
`TIPS` represents for?

Reply:
some tips information for  the collector process name

Reply:
OK. Then I proposal go for `XXXXX_NAME_TAG`

Reply:
Why do you use `tag`?  I think that  the word `org.apache.skywalking.apm.collector.boot.CollectorBootStartUp` is the prompt character  of collector process information, and I think  the `tag` word  mean that it is  attribute filed of collector process.

Reply:
That is the `TAG` for process identification.

Reply:
The `grep` command just find the character,  not for `tag`.  So I don't think the `TAG` is an good idea.

Reply:
ok，just use `XXXXX_NAME_TAG`